hack/make-rules/test-cmd-util.sh

```diff
@@ -2824,6 +2824,18 @@ run_deployment_tests() {
   # Clean up
   kubectl delete deployment nginx "${kube_flags[@]}"
 
+  ### Show the status of the rollout
+  # Pre-condition: no deployment exists
+  kube::test::get_object_assert deployment "{{range.items}}{{$id_field}}:{{end}}" ''
+  # create namespace 'my-namespace'
+  kubectl create namespace my-namespace "${kube_flags[@]}"
+  kubectl config set-context "${CONTEXT}" --namespace="my-namespace"
+  kube::test::get_object_assert 'namespace my-namespace' "{{$id_field}}" 'my-namespace'
+  kubectl create -f hack/testdata/deployment-rollout-status.yaml "${kube_flags[@]}"
+  kubectl rollout status -f hack/testdata/deployment-rollout-status.yaml "${kube_flags[@]}"
+  # Clean up
+  kubectl delete deployment nginx "${kube_flags[@]}"
+  
```

Comment:
@mengqiy i am no idea about this test, can help me fix the test fail? appreciate

Reply:
I think test failure is caused by `run_deployment_tests` has called `create_and_use_new_namespace` at the beginning of the test. 
So I believe there is a `--namespace` flag in `"${kube_flags[@]}"` that may override your flag specified in the file.

You may want to move your new test to a new test case name that doesn't call `create_and_use_new_namespace`.